quinn-proto/src/frame.rs

```diff
@@ -826,6 +826,29 @@ impl Datagram {
     }
 }
 
+/// Indicates whether a frame of type `F` needs to be transmitted
+///
+/// This type is around `bool` which uses the `#[must_use]` attribute in order
```

Comment:
This sentence reads a bit odd to me, maybe "This type wraps around `bool` and uses the ..."?